cfpb/cfgov-refresh
# Removing oah email popup

Removing oah email popup code and styles.

## Additions
- Moved `throttle.js` function from `cfgov/unprocessed/js/modules/util/email-popup-helpers.js` to it's own file.

## Removals

- Removed templates, styles, and logic associated with the oah popup.

## Changes

- Modified `cfgov/unprocessed/css/molecules/autocomplete.less` to fix issue with search input width.
- Modified `cfgov/jinja2/v1/_includes/organisms/search-bar.html`  to add input class. 


## Testing

1. Run `gulp styles.`
2. Visit `http://localhost:8000/owning-a-home/` and enter `example@test`. 
3. Hit enter and you should see the alert notification
4. Run `python cfgov/manage.py update_index`
5. 2. Visit `http://localhost:8000/ask-cfpb/`
6. Verify that the search looks as expected.
7. Type `car` in the search field and verify that there is a dropdown.

## Screenshots

-  
<img width="463" alt="screen shot 2017-07-11 at 8 04 38 pm" src="https://user-images.githubusercontent.com/1696212/28096077-361b2d36-6674-11e7-9acd-2a1700b6f4ed.png">

-
<img width="1247" alt="screen shot 2017-07-11 at 8 08 50 pm" src="https://user-images.githubusercontent.com/1696212/28096183-c84ee42c-6674-11e7-9aa2-69eca366d821.png">

- 
<img width="550" alt="screen shot 2017-07-11 at 8 09 21 pm" src="https://user-images.githubusercontent.com/1696212/28096192-e501adac-6674-11e7-9d8a-027779f8329d.png">


## Notes

-

## Todos

-

## Checklist
